tests/lists.bats

```diff
@@ -180,3 +180,20 @@ IMAGE_LIST_S390X_INSTANCE_DIGEST=sha256:882a20ee0df7399a445285361d38b711c299ca09
     # Make sure we can add the new image to the list.
     run_buildah manifest add test-list $(< ${TESTDIR}/image-id.txt)
 }
+
+@test "manifest-add-to-list-from-storage" {
+    run_buildah pull --arch=amd64 busybox
+    run_buildah tag busybox test:amd64
+    run_buildah pull --arch=arm64 busybox
+    run_buildah tag busybox test:arm64
+    run_buildah manifest create test
+    run_buildah manifest add test test:amd64
+    run_buildah manifest add --variant=v8 test test:arm64
+    run_buildah manifest inspect test
+    # must contain amd64
+    expect_output --substring "amd64"
+    # must contain arm64
+    expect_output --substring "arm64"
+    # must contain variant v8
+    expect_output --substring "v8"
```

Comment:
This check will stop being meaningful at some point, as the image we're added to the manifest will eventually be marked with variant=v8 in its image configuration (to confirm: `skopeo --override-arch arm64 inspect --config --raw docker://docker.io/library/busybox` -- the registries.conf we're using redirects us to quay.io/libpod/busybox, which is an older version that didn't include that field in its configuration, but I don't expect that to be the case forever).  I'd suggest specifying a value that's unlikely to be in the image on line 191, and then checking for that here.

Reply:
I agree changed it to arbitrary variant and fixed in latest commit.